easybuild/easyblocks/p/python.py

```diff
@@ -437,6 +450,18 @@ def build_step(self, *args, **kwargs):
 
         super(EB_Python, self).build_step(*args, **kwargs)
 
+    def test_step(self):
+        """Test Python build via 'make test'."""
+        # Turn on testing by default for recent Python versions
+        relevant_python = (self.name.lower() == 'python' and LooseVersion(self.version) >= LooseVersion('3.10'))
```

Comment:
I did it like this as this easyblock is inherited by Tkinter and I wanted to restrict the change in default behaviour